### DISCUSSION TITLE
Example structure

### DISCUSSION THREAD

**Discussion by CarrieBrown** [LukeCarrier/mkdocs-theme-bootstrap4#45] [Q&A] [answered]
Hello, I am trying to implement your theme but am having problems with forcing naming in the top menu bar. Can you provide an example on how to specify custom names for the menu options beyond the file name?

For example, I have a folder entitled "01_GettingStarted" but I would like it to show up as only "Getting Started" on the live site. With previous themes, we used an index.md file with a header where you could specify things like `title` and `weight`. Is this supported with bootstrap4? And if so, how do we implement it?

It would be particularly helpful to view the file structure behind the example screenshot shown in the README.md.

Any guidance you can provide would be appreciated!

**Comment by LukeCarrier** [comment, marked as answer]
Hi @CarrieBrown,

Off the top of my head the titles shown in navigation come from (in descending order of precedence):

- The title portion of the entry in the `nav` entry in `mkdocs.yml` (e.g. `- My Title: some/page/index.md`).
- The first level one heading in the referenced document (`# My Title`).

I don't know if MkDocs looks at the frontmatter for titles by default, but I'm fairly sure it doesn't do this for weighting.

Hope this helps!
